docs/contribution/community-components.md

```diff
@@ -0,0 +1,299 @@
+to contribute regardless of the completion. By creating a single environment for
+this we hope to connect contributors who need a designer or developer to help
+out. Part of our effort in helping contributors who don't have the means to
+complete the checklist includes displaying these contribution on our website
+gallery as `code only` or `design only` assets.
+
+**Can I contribute to `code only` or `design only` assets?**
+
+YES! We encourage you to combine efforts on these components. Our main goal is
+to have a community lead library. If you're a designer interested in
+contributing to `code only` assets, reach out to the codeowner to work with
+them. If you're a developer interested in contributing to `design only` assets,
+reach out to the designer to let them know you're interested in building their
+component; you should find that all of the design assets needed for development
+already live in the package directory and all they're missing is code.
+
+**If my contribution is `code only`/`design only`, do I leave my initial
+contribution issue open when I merge my PR?**
+
+No. Please close the issue so that Carbon knows that you already have a package
+directory assigned. Instead, open a `need help` issue, and add a label depending
+on if you need a dev or design contributor to help (`need dev`/ `need design`).
+You can reference your original issue in this issue so that potential
+contributors can see the history.
+
+**How does the component status on the website gallery card work? (draft, beta,
+stable)**
+
+The component status is a self assigned status. This does not depend on how many
+of the design or development guidelines you've checked off. You can consider
+your component stable even if you haven't met all of the guidelines. The purpose
+of the component status is to show the community whether your component will be
+undergoing major changes, is still undergoing testing or is ready to be used.
+
+**Why do contributions have issues and website cards?**
+
+The Carbon community has a variety of users, which means that people work in
+different spaces and use different tools. We have found that even though
+developers live on GitHub and are accustomed to opening and checking issues,
+designers often times don't find that as useful for learning about existing
+efforts. Part of our vision for this community components model was to make
+contributing and learning how/what to contribute easier. Having an issue and a
+website card with useful information (i.e. draft/beta/stable status,
+`code only`/`design only`) will allow more people to learn about existing
+efforts and how to help than if this information only lived on the website or on
+GitHub.
+
+**How do I include my design assets in the component package?**
+
+If you're contributing design, you will add your sketch files to the community
+components [box folder](url here). In the monorepo, we will include the
```

Comment:
has this folder been created yet?

Reply:
No. We haven't created any assets for this. We wanted to get feedback on the model first.

Reply:
ok got it, I assumed we had some existing from our previous addons